.gitignore

```diff
@@ -24,10 +24,12 @@
 
 # Sensitive files
 /config/database.yml
+/config/local_env.yml
```

Comment:
So is this still needed? (and `/vendor/bundle` as well?)

Reply:
No, they are deleted now